src/components/Equation.js

```diff
@@ -51,6 +52,18 @@ export default class Equation extends React.Component {
             onChange={this.handleChange}
           />
         );
+      } else if (paramConfig.type === "select") {
+        attribs.push(
+          <SelectInput
+            id={param}
+            label={paramConfig.label}
+            key={param}
+            fieldKey={param}
+            options={paramConfig.options}
+            value={this.state[param]}    
+            onChange={this.handleChange}
+          />
+        )
```

Comment:
OFF TOPIC: This is making me think I'm going to change that PR to be a switch that returns a Jsx tag, which. you will in turn pass values to? what do you think?   A Tag to make tags seems a bit weird.   So this method will look like (pseudo code):

```
const Field = switch(paramConfig.type) {
case "currency": Currency
case "select": SelectInput
...
}

attribs.push(<Field id={param}  ... )
```

Does that make sense?

Reply:
Yeah - did you see my feedback on that PR? We can chat through it if you like